test/routable_component.js

```diff
@@ -63,75 +63,75 @@ describe('RoutableComponent', function() {
       });
     });
   });
 
-  /** @test {RoutableComponent#register_component} */
-  context('register_component', function() {
-    subject(function() { return new RoutableComponent({}, this.routes, this.controller_classes) });
+  /** @test {RoutableComponent#registerComponent} */
+  context('registerComponent', function() {
+    subject(function() { return new RoutableComponent({}, this.routes, this.controllerClasses) });
 
     it('surely registered', function() {
-      this.subject.register_component('MyEventSourceComponent', new MyEventSourceComponent());
+      this.subject.registerComponent('MyEventSourceComponent', new MyEventSourceComponent());
       assert(this.MyEventSourceComponent instanceof MyEventSourceComponent);
       this.MyEventSourceComponent.emit('myevent', "1", 2),
       assert(this.subject.controllers.MyController.component === this.subject);
-      assert.deepEqual(this.subject.controllers.MyController.myaction_args, ["1", 2]);
+      assert.deepEqual(this.subject.controllers.MyController.myactionArgs, ["1", 2]);
     });
 
     it('surely registered with another name', function() {
-      this.subject.register_component('AnotherNameComponent', new MyEventSourceComponent());
+      this.subject.registerComponent('AnotherNameComponent', new MyEventSourceComponent());
       assert(this.subject.components.AnotherNameComponent instanceof MyEventSourceComponent);
       this.subject.components.AnotherNameComponent.emit('myevent', "1", 2),
       assert(this.subject.controllers.MyController === undefined);
     });
 
     it('registers only one component event', function() {
       const component = new MyEventSourceComponent();
-      this.subject.register_component('MyEventSourceComponent', component);
-      this.subject.register_component('MyEventSourceComponent', component);
+      this.subject.registerComponent('MyEventSourceComponent', component);
+      this.subject.registerComponent('MyEventSourceComponent', component);
       this.MyEventSourceComponent.emit('plus');
       assert(this.subject.controllers.MyController.count === 1);
     });
   });
 
-  /** @test {RoutableComponent#unregister_component} */
-  context('unregister_component', function() {
-    subject(function() { return new RoutableComponent({}, this.routes, this.controller_classes) });
+  /** @test {RoutableComponent#unregisterComponent} */
+  context('unregisterComponent', function() {
+    subject(function() { return new RoutableComponent({}, this.routes, this.controllerClasses) });
 
     it('surely unregistered', function() {
       const component = new MyEventSourceComponent();
-      this.subject.register_component('MyEventSourceComponent', component);
+      this.subject.registerComponent('MyEventSourceComponent', component);
       this.MyEventSourceComponent.emit('plus');
-      this.subject.unregister_component('MyEventSourceComponent');
+      this.subject.unregisterComponent('MyEventSourceComponent');
```

Comment:
__[eslint]__
* Error - Unexpected 'this'. (no-invalid-this)

<!-- Comment by [SideCI](https://sideci.com) -->